docs/navigation-configuration.md

```diff
@@ -66,6 +74,23 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 - **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
   **nodeAccessibilityResolver** receives all values defined in the node configuration. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js) for the **constraints** example.
 
+## Context Switcher
```

Comment:
`### Context switcher` (just the first letter of the first word is capitalised, and the heading should be at level 3).

I would go with `context switcher` rather than `Context Switcher`.

Reply:
Changed everywhere